src/components/main-form.js

```diff
@@ -94,13 +93,90 @@ const InputWrapper = styled('div')`
   }
 `
 
-const ColHeader = styled(Col)`
-  font-weight: bold;
+const AmortizationTable = styled(Table)`
+  margin-top: 30px;
+
+  th,
+  tbody {
+    text-align: center;
+  }
+
+  th:first-of-type {
+    border: 0;
+  }
+
+  th:nth-child(6),
+  td:nth-child(6) {
+    border-left-width: thick;
+    border-left-color: ${DARK_GRAY}};
+  }
 `
 
+const centered = css`
+  margin: 0 auto;
+  display: block;
+`
+
+const highlightTableCells = (year, fv1, fv2) => {
+  return (
+    (year + 1) % 15 === 0 &&
+    css`
+      td:nth-child(n + 2):nth-child(-n + 5) {
+        background-color: ${fv1 > fv2 ? GREEN : LIGHT_GRAY};
+      }
+
+      td:nth-child(n + 6):nth-child(-n + 9) {
+        background-color: ${fv2 > fv1 ? GREEN : LIGHT_GRAY};
+      }
+    `
+  )
+}
+
+const hoverTableCells = year => {
+  // grab table cells depending on year
+  const tableCells = Array.from(
+    document.getElementsByTagName('tbody')[0].children[year].children
+  )
+
+  const hover = (color, brightness, cell, index) => {
+    const nextCells = tableCells[index + 4]
+    const prevCells = tableCells[index - 4]
+
+    cell.style.backgroundColor = index !== 0 && color
+    cell.style.filter = `brightness(${brightness}%)`
+
+    if (typeof nextCells !== 'undefined' && index !== 0) {
+      nextCells.style.backgroundColor = color
+      nextCells.style.filter = `brightness(${brightness}%)`
+    }
+
+    if (typeof prevCells !== 'undefined' && index !== 4) {
+      prevCells.style.backgroundColor = color
+      prevCells.style.filter = `brightness(${brightness}%)`
+    }
+  }
+
+  // iterate over all cells in row and set appropriate color depending on mouse event
+  tableCells.forEach((cell, index) => {
+    // do not set background color for rows that already have a background color
+    if ((year + 1) % 15 === 0) {
+      cell.addEventListener('mouseover', () => hover(null, 80, cell, index))
+      cell.addEventListener('mouseleave', () => hover(null, 100, cell, index))
+    } else {
+      cell.addEventListener('mouseover', () =>
+        hover(LIGHT_GRAY, 80, cell, index)
+      )
+      cell.addEventListener('mouseleave', () =>
+        hover('white', 100, cell, index)
+      )
+    }
+  })
+}
```

Comment:
Not really sure what this looks like. Wouldn't https://reactstrap.github.io/components/tables/#app get us most of the way there?

Reply:
no, the reacstrap tables (which I am already using here) don't provide the hovering functionality here.